macros/base/probing/overrides/beacon_probe_overrides.cfg

```diff
@@ -0,0 +1,79 @@
+#############################################################################
+# Probe macros base on the defaults but modified with Beacon specific improvements
+# Improvements are from the ANNEX Discord and allow for QGL/Tilt in scan mode
+#############################################################################
+
+[gcode_macro FAST_QUAD_GANTRY_LEVEL]
```

Comment:
I'm not a fan of using a new `FAST_...` macro for a duplication of the same feature that QGL already provide but adapted to be faster. I'm pretty sure everyone will always want to use the fast version.

So why not merging this in the original overrides files with a check on the `beacon_contact` (and also standard `beacon`?) probe type to do the fast version?
`https://github.com/Frix-x/klippain/blob/main/macros/base/probing/overrides/qgl.cfg`
`https://github.com/Frix-x/klippain/blob/main/macros/base/probing/overrides/z_tilt.cfg`

This way, when running the standard QGL/Z_TILT macro that is already available in the frontends, it will always work for everyone, using the fast way if you can go fast with a beacon.